packages/react/src/RelativeTime/RelativeTime.docs.json

```diff
@@ -101,6 +101,12 @@
       "name": "ref",
       "type": "React.RefObject<RelativeTimeElement>"
     },
+    {
+      "name": "noTitle",
+      "type": "boolean",
+      "defaultValue": "",
+      "description": "Removes the `title` attribute provided on the element by default"
```

Comment:
Hi! 👋 

Can you also add when is it recommended to skip title?

Reply:
In this description? Or are you talking about updating the [in-progress docs PR](https://github.com/primer/design/pull/795/files)?

Reply:
Ideally both places, but either is fine for now :)